### PR TITLE
Tv/platf 1823 correct polygon os shared storefront metdata

### PR DESCRIPTION
**Pull request by tv3636**

### CODE REVIEW

File: src/parsers/simplehash.js [hunk 1]
```diff
@@ -10,10 +10,13 @@ export const parse = (asset) => {
   } = asset.extra_metadata;
 
   return {
-    contract: asset.contract_address,
+    contract: _.toLower(asset.contract_address),
     tokenId: asset.token_id,
     name: asset.name,
     collection: _.toLower(asset.contract_address),
+    slug: asset.collection.marketplace_pages.filter(
```
Comment (nofir) [inline src/parsers/simplehash.js:17]: im not sure why is that needed.

**Review by nofir — CHANGES_REQUESTED** [review]

File: src/parsers/simplehash.js (hunk 1 above)
Comment (tv3636) [reply src/parsers/simplehash.js:17]: we use the slug for the collection id in extended metadata with opensea shared contract: https://github.com/reservoirprotocol/metadata/blob/main/src/extend/opensea-shared-contract/index.js#L10

so without this all the tokens on the shared contract will be assigned to collectionId: `0x2953399124f0cbb46d2cbacd8a89cf0599974963:opensea-undefined`

**Review by nofir — APPROVED** [review]
